packages/x64dbg.dbgchild.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,62 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+VM-Remove-PreviousZipPackage ${Env:chocolateyPackageFolder}
+
+try {
+    $toolSrcDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+    $toolDir = Join-Path ${Env:RAW_TOOLS_DIR} 'x64dbg\release' -Resolve
+
+    $packageArgs = @{
+        packageName   = ${Env:ChocolateyPackageName}
+        unzipLocation = $toolSrcDir
+        url           = 'https://github.com/therealdreg/DbgChild/releases/download/beta10/DbgChild.Beta.10.zip'
+        checksum      = 'f17f588795d8f5f94d71335a8acfa58946bb03a94a5637be7f3e804c652ea2b4'
+        checksumType  = 'sha256'
+    }
+    Install-ChocolateyZipPackage @packageArgs
+    $toolSrcDir = Join-Path $toolSrcDir 'DbgChild Beta 10'
+    VM-Assert-Path $toolSrcDir
+
+    $archs = @("x32", "x64")
+    foreach ($arch in $archs) {
+        $archDstDir = Join-Path $toolDir "${arch}" -Resolve
+        $pluginDstDir = Join-Path $archDstDir 'plugins'
+        if (-Not (Test-Path $pluginDstDir -PathType Container)) {
+            New-Item -ItemType directory $pluginDstDir -Force -ea 0 | Out-Null
+        }
+        VM-Assert-Path $pluginDstDir
+
+        # Move 32/64-bit plugin DLL itself into the arch directory
+        $pluginSrcPath = Join-Path $toolSrcDir "release\${arch}\plugins" -Resolve
+        Get-ChildItem -Path $pluginSrcPath -File | Move-Item -Destination $pluginDstDir -Force -vb
```

Comment:
What are the `-vb` at the end of these lines?

Reply:
the `-Verbose` common parameter
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_commonparameters?view=powershell-7.3
it's sometimes helpful for debugging, but I'll remove for production now